website/src/data/users.js

```diff
@@ -334,6 +334,14 @@ const Users = [
     source: 'https://github.com/eta-dev/eta',
     tags: ['opensource'],
   },
+  {
+    title: "Evan Tay's Personal Website",
+    description: "A Software Engineer's blog, documentation and project portfolio.",
+    preview: require('./showcase/evantay.png'),
+    website: 'https://evantay.com/',
+    source: 'https://github.com/DigiPie/kaya-folio',
+    tags: ['design', 'opensource', 'personal'],
```

Comment:
```suggestion
    tags: ['opensource', 'personal'],
```

Removing "design" as I don't think your site stand out by its design compared to other sites having this tag